Dangerfile

```diff
@@ -0,0 +1,13 @@
+# Sometimes it's a README fix, or something like that - which isn't relevant for
+# including in a project's CHANGELOG for example
+declared_trivial = github.pr_title.include? "#trivial"
+
+# Make it more obvious that a PR is a work in progress and shouldn't be merged yet
+warn("PR is classed as Work in Progress") if github.pr_title.include? "[WIP]"
+
+# Warn when there is a big PR
+warn("Big PR") if git.lines_of_code > 500
+
+# Don't let testing shortcuts get into master by accident
+fail("fdescribe left in tests") if `grep -r fdescribe spec/ `.length > 1
```

Comment:
should these not be `>=` or `>0` ?

Reply:
@willhall88 it's oddly returning true for either. See below for details.

```
[5] pry(main)> `grep -r "RSpec.describe ParentCategory" spec/`
=> "spec//domain/parent_category_spec.rb:RSpec.describe ParentCategory do\n"
[6] pry(main)> `grep -r "RSpec.describe ParentCategory" spec/`.length > 0
=> true
[7] pry(main)> `grep -r "RSpec.describe ParentCategory" spec/`.length > 1
=> true
```

This PR does not need to be reviewed just yet.